custom/icds_reports/utils/__init__.py

```diff
@@ -1194,7 +1205,7 @@ def create_thr_report_excel_file(excel_data, data_type, month, aggregation_level
     headers = ["S.No"]
     main_headers = ['State', 'District', 'Block', 'Sector', 'Awc Name', 'AWW Name', 'AWW Phone No.',
                    'Total No. of Beneficiaries eligible for THR',
-                   'Total No. of beneficiaries received THR in given month',
+                   f'Total No. of beneficiaries received THR {thr_days_info} in given month',
```

Comment:
Maybe doesn't matter, but I think this will have a double space instead of a single one if the variable doesn't get overwritten. All the workarounds I can think of would result in uglier code.